### PR TITLE
Lit 3.0 pre-releases blog post

### PR DESCRIPTION
**Pull request by justinfagnani**

### CODE REVIEW

File: packages/lit-dev-content/site/blog/2023-05-15-lit-3.0-prerelease.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+```json
+  "lit": "^3.0.0-pre.0"
+  "lit-html": "^3.0.0-pre.0"
+  "lit-element": "^4.0.0-pre.0"
+  "@lit/reactive-element": "^2.0.0-pre.0"
+```
+
+You can also use the `pre` npm tag, like `npm i lit@pre`.
+
+All other packages, like labs packages, have pre-release versions too that depend on the prerelease core libraries. If you depend on those you'll have to update them too. If you're more daring you can use npm overrides to select the pre-releases even for dependencies. This should work for most dependencies.
```
Comment (augustjk) [inline packages/lit-dev-content/site/blog/2023-05-15-lit-3.0-prerelease.md:48]: I see more pre-release than prerelease so I think we'll go with the hyphenated version consistently?
```suggestion
All other packages, like labs packages, have pre-release versions too that depend on the pre-release core libraries. If you depend on those you'll have to update them too. If you're more daring you can use npm overrides to select the pre-releases even for dependencies. This should work for most dependencies.
```

File: packages/lit-dev-content/site/blog/2023-05-15-lit-3.0-prerelease.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+Even if dependencies are using Lit 2.x, the good news is that Lit 2.x and Lit 3.x are interoperable, because:
+1. The inherent interoperability web components: components built with different libraries work together, including those build with different versions of Lit.
+2. We made interop of core features like templates and directives a priority for Lit 2. You can share templates, directives, decorators, etc., across Lit versions.
+
+## Docs
+
+We are preparing new docs for 3.0 on [lit.dev](https://lit.dev). Even though these will be mostly the same as 2.x, we are clarifying the browser and toolchain support, and want to make it easy to select the right docs set for the version you're using to enable future changes specific to to 3.x. At the 3.0 launch, we’ll archive the 2.x docs (but they will remain available in the doc version dropdown, as will 1.x). During the 3.0 prerelease phase, 2.x will remain the default, and you select 3.0 from the dropdown next to the Documentation tab.
```
Comment (augustjk) [inline packages/lit-dev-content/site/blog/2023-05-15-lit-3.0-prerelease.md:56]:
```suggestion
We are preparing new docs for 3.0 on [lit.dev](https://lit.dev). Even though these will be mostly the same as 2.x, we are clarifying the browser and toolchain support, and want to make it easy to select the right docs set for the version you're using to enable future changes specific to to 3.x. At the 3.0 launch, we’ll archive the 2.x docs (but they will remain available in the doc version dropdown, as will 1.x). During the 3.0 pre-release phase, 2.x will remain the default, and you select 3.0 from the dropdown next to the Documentation tab.
```

**Review by augustjk — APPROVED** [review]

File: packages/lit-dev-content/site/blog/2023-05-15-lit-3.0-prerelease.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+The Lit 3.0 changes are mostly in browser support, removing deprecated APIs, and how publish packages. If you run Lit 2.x with no deprecation warnings, this should be a seamless upgrade!
+
+Here are the biggest things Lit 3.0 changes:
+- IE11 is no longer supported.
+- Lit's npm modules are now published as ES2021.
+- APIs deprecated during the Lit 1.x release timeframe have been removed.
+- SSR hydration support modules were moved to the `@lit-labs/ssr-client` package.
+
+Detailed change logs can be found [on GitHub](https://github.com/lit/lit/releases?q=%22-pre.0%22&expanded=true).
```
Comment (AndrewJakubowicz) [inline packages/lit-dev-content/site/blog/2023-05-15-lit-3.0-prerelease.md:31]: Worth also linking to [`/docs/v3/releases/upgrade`](https://pr1116-d13180f---lit-dev-5ftespv5na-uc.a.run.app/docs/v3/releases/upgrade/) to navigate readers to the pre-release upgrade guide page?

**Review by AndrewJakubowicz — APPROVED** [review]

**Comment by augustjk** [comment]
Moving base branch to `version-v3-doc` so we can release the v3 docs and blog at the same time with their cross references.

File: packages/lit-dev-content/site/blog/2023-05-15-lit-3.0-prerelease.md (hunk 3 above)
Comment (augustjk) [reply packages/lit-dev-content/site/blog/2023-05-15-lit-3.0-prerelease.md:33]: Added!

**Pull request merged**